java/vector/src/main/java/org/apache/arrow/vector/ipc/file/ArrowFileWriter.java

```diff
@@ -16,14 +16,18 @@
  * limitations under the License.
  */
 
-package org.apache.arrow.vector.file;
+package org.apache.arrow.vector.ipc.file;
```

Comment:
What do you feel about get rid of "file" and "stream" sub namespace, i.e.

```
org.apache.arrow.vector.ipc.ArrowFileWriter
```
```
org.apache.arrow.vector.ipc.ArrowStreamWriter
```
I think these two namespaces `file` and `stream` are not very complicated, they can probably be combined